app/processing/is-agreement-closed.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../data')
+
+const isAgreementClosed = async (paymentRequest) => {
+  const schemeId = paymentRequest.schemeId
+  const frn = paymentRequest.frn
+  const agreementNumber = paymentRequest.agreementNumber
+  const todaysDate = new Date()
```

Comment:
Suggest using `currentDate` just because that's the convention used elsewhere in the code.

Reply:
renamed